webapi/src/main/scala/org/knora/webapi/core/AppServer.scala

```diff
@@ -212,6 +216,9 @@ object AppServer {
    */
   val testWithoutSipi: ZLayer[AppServerEnvironment, Nothing, Unit] =
     ZLayer {
-      startup(false, false)
+      for {
+        appServer <- AppServer.init()
+        _         <- appServer.start(requiresRepository = false, requiresIIIFService = false)
+      } yield ()
     }
```

Comment:
the name of the parameter was misleading, I changed it to `requiresAdditionalRepositoryChecks` because it needs the repository but in these cases it is initialized beforehand, so there is no need to update data or load the ontology cache.